docs/1___overview.adoc

```diff
@@ -67,8 +67,12 @@ The interface types have large parts in common, defined in <<fmi-common-concepts
 In particular:
 
 * FMI Application Programming Interface \(C) -- <<general-mechanisms>> +
-All required equations or tool coupling computations are evaluated by calling standardized C functions.
+All required equations or tool coupling computations are evaluated by calling standardized C functions from the importer into the FMU.
+The FMU can signal certain events back to the importer using callback functions provided by the importer.
 C is used because it is the most portable programming language today and is the only programming language that can be utilized in all embedded control systems.
+The FMI API does not restrict what operating system services an FMU can use on the platform it runs on.
+However, for maximal portability, any dependency on the target platform should be minimized and operating system services should be accessed only through standard libraries.
```

Comment:
It think this helps a lot!

So as you said in the meeting you decided not to add a hint that an FMU that is compiled or can be compiled with gcc and only depends on the GNU Libc is likely supported by most linux based FMU importers. Okay.

Reply:
Yes, I think llvm and hopefully future initiatives like that will improve the situation that gcc is currently in (dependencies on certain version of stdlib etc.) - enshrining gcc seems the wrong hint here.